_posts/2022-06-08-filesystem-integrity.adoc

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Enabling Integrity on Filesystem realms"
+date: 2022-06-08
+tags: filesystem-realm integrity
+synopsis: An overview of the new support for filesystem realm integrity verification.
+author: araskar
+link:
+---
+
+Previously, any data in the filesystem realm was not verified, this meant that anyone with access to the identity files, could tamper with the data without any way of knowing. Now filesystem integrity has been added by generating a signature through the use of a public-private key pair.
+
+== An overview of the new attributes
+
+The filesystem realm now supports the following attributes
+
+* ``key-store``: This attribute specifies the key store resource where the key pair resides in. This key store must be configured prior to creating the filesystem realm. This attribute is optional and if not specified, the filesystem realm will not verify its integrity.
+
+* ``key-store-alias``: This attribute specifies the alias to the key pair in the key store to verify filesystem integrity. This attribute is required if the `key-store` attribute is specified.
```

Comment:
Was just reviewing your documentation update and might be good to use the descriptions of the attributes from there here as well.